.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Onboarding
+about: 'Template for onboarding new developers '
+title: Onboarding to Azul
+labels: ''
+assignees: ''
+
```

Comment:
```suggestion
```

Reply:
This was auto-generated by github; the space has been removed, might come back if the template is modified within the github UI.